odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig

```diff
@@ -21,7 +22,16 @@ BR2_LINUX_KERNEL_INTREE_DTS_NAME="broadcom/bcm2711-rpi-4-b"
 BR2_LINUX_KERNEL_NEEDS_HOST_OPENSSL=y
 BR2_PACKAGE_LINUX_TOOLS_GPIO=y
 BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
-BR2_PACKAGE_XZ=y
+BR2_PACKAG
```

Comment:
Is this a typo?

Reply:
I have no idea how but it seems to be a result of the merge.  That package not being there would be very bad.

Reply:
Might've been my bad :grimacing:

Reply:
Also this file should be deleted: odysseus/odysseus_tree/overlays/rootfs_overlay_common/etc/modules

Ill commit a fix.

Reply:
Any chance you can add back in?

Reply:
Thank you!

Reply:
All right should be all set.  Ig github being weird.